nextstrain/cli/command/update.py

```diff
@@ -116,7 +115,7 @@ def run(opts):
     # Check our own version for updates
     print(heading(f"Checking for newer versions of Nextstrain CLI…"))
     print()
-    newer_version = check_for_new_version()
+    newer_version, update_instructions = check_for_new_version()
 
     # Perform updates
     if not updates:
```

Comment:
If there are no runtime/pathogen updates to perform, then the update instructions for the CLI itself is never printed.